src/pages/home/HeaderView.js

```diff
@@ -63,7 +63,7 @@ const HeaderView = props => (
                         onPress={() => {
                             const {participants} = props.report;
                             if (participants.length === 1) {
-                                Navigation.navigate(ROUTES.getProfileRoute(participants[0]));
```

Comment:
I know this is kinda unrelated to your changes, but it looks like `participants` is a duplicate name from the upper scope (line 53). Is there any reason we're re-accessing/redefining a variable for `props.reports.participants` here?

Reply:
You're right that there's no need to redefine this here and actually it convenient got changed/resolved after merging master.